parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -271,143 +271,91 @@ function installClearContainersRuntime() {
 }
 
 function installGo() {
 	export GO_SRC=/usr/local/go
 	export GOPATH="${HOME}/.go"
+
 	if [[ -d "$GO_SRC" ]]; then
 		rm -rf "$GO_SRC"
 	fi
 	if [[ -d "$GOPATH" ]]; then
 		rm -rf "$GOPATH"
 	fi
 
-    retrycmd_if_failure_no_stats 180 1 5 curl -fsSL https://golang.org/VERSION?m=text > /tmp/gover.txt
-    GO_VERSION=$(cat /tmp/gover.txt)
-    retrycmd_get_tarball 60 1 /tmp/golang.tgz https://storage.googleapis.com/golang/${GO_VERSION}.linux-amd64.tar.gz
-    tar -v -C /usr/local -xzf /tmp/golang.tgz
+	retrycmd_if_failure_no_stats 180 1 5 curl -fsSL https://golang.org/VERSION?m=text > /tmp/gover.txt
+	GO_VERSION=$(cat /tmp/gover.txt)
+	echo "Installing Go version $GO_VERSION..."
 
-	export PATH="${GO_SRC}/bin:${PATH}:${GOPATH}/bin"
-}
+	retrycmd_get_tarball 60 1 /tmp/golang.tgz https://storage.googleapis.com/golang/${GO_VERSION}.linux-amd64.tar.gz
+	tar -v -C /usr/local -xzf /tmp/golang.tgz
+	rm -rf "/tmp/golang.tgz" "/tmp/gover.txt"
 
-function buildRunc() {
-	# Clone the runc source
-	echo "Cloning the runc source..."
-	mkdir -p "${GOPATH}/src/github.com/opencontainers"
-	(
-	cd "${GOPATH}/src/github.com/opencontainers"
-	git clone "https://github.com/opencontainers/runc.git"
-	cd runc
-	git reset --hard v1.0.0-rc4
-	make BUILDTAGS="seccomp apparmor"
-	make install
-	)
-
-	echo "Successfully built and installed runc..."
+	export PATH="${GO_SRC}/bin:${PATH}:${GOPATH}/bin"
 }
 
-function buildCRIO() {
-	# Add CRI-O repositories
-	echo "Adding repositories required for cri-o..."
-	add-apt-repository -y ppa:projectatomic/ppa
-	add-apt-repository -y ppa:alexlarsson/flatpak
-	apt-get update
-
-	# Install CRI-O dependencies
-	echo "Installing dependencies for CRI-O..."
-	apt-get install --no-install-recommends -y \
+function buildContainerd() {
+	apt-get update && apt-get install --no-install-recommends -y \
 		btrfs-tools \
 		gcc \
-		git \
 		libapparmor-dev \
-		libassuan-dev \
 		libc6-dev \
-		libdevmapper-dev \
-		libglib2.0-dev \
-		libgpg-error-dev \
-		libgpgme11-dev \
-		libostree-dev \
 		libseccomp-dev \
-		libselinux1-dev \
 		make \
-		pkg-config \
-		skopeo-containers
+		pkg-config
 
 	installGo;
 
-	# Install md2man
-	go get github.com/cpuguy83/go-md2man
-
-	# Fix for templates dependency
-	(
-	go get -u github.com/docker/docker/daemon/logger/templates
-	cd "${GOPATH}/src/github.com/docker/docker"
-	mkdir -p utils
-	cp -r daemon/logger/templates utils/
-	)
-
-	buildRunc;
-
-	# Clone the CRI-O source
-	echo "Cloning the CRI-O source..."
-	mkdir -p "${GOPATH}/src/github.com/kubernetes-incubator"
+	echo "Cloning the cri-containerd source..."
+	mkdir -p "${GOPATH}/src/github.com/containerd"
 	(
-	cd "${GOPATH}/src/github.com/kubernetes-incubator"
-	git clone "https://github.com/kubernetes-incubator/cri-o.git"
-	cd cri-o
-	git reset --hard v1.0.0
+	cd "${GOPATH}/src/github.com/containerd"
+	git clone "https://github.com/containerd/cri.git"
+	cd cri
+	git reset --hard "v1.0.0-rc.0"
```

Comment:
I think it would be better to install from `github.com/containerd/containerd.git` v1.1.0-rc.0.

`github.com/containerd/cri.git` is a containerd plugin now, and the make logic in that repo is mainly for CI testing. :)

But of course you could still use the systemd files. :)

Reply:
oh gotcha :)

Reply:
👍

Reply:
fixed it here: https://github.com/Azure/acs-engine/pull/2547